orcid-persistence/src/main/java/org/orcid/persistence/aop/ProfileLastModifiedAspect.java

```diff
@@ -130,7 +127,7 @@ public void updateLastModifiedDateAndIndexingStatus(String orcid) {
         Date last = retrieveLastModifiedDate(orcid);
         LastModifiedMessage mess = new LastModifiedMessage(orcid,last);
         if (messaging.isEnabled()){
-            if(!messaging.send(mess,JmsDestination.UPDATED_ORCIDS)) {
+            if(!messaging.send(mess, JmsDestination.UPDATED_ORCIDS)) {
```

Comment:
This line sends a message to the queue whenever a record is updated.  Coupled with the code you added in OrcidProfileManagerImpl.java this means messages are being sent twice